### ISSUE TITLE
Unable to add ALT text to items within an item set

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#70]
> **Migrated from GitLab**
> **Original Author:** Lenszie Vilfranche (`@vilfranchel`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/416

## Expected behaviour

We should have the ability to add alt text to items within an item set. 

## Current behaviour

When I go to the item sets, I am able to add/edit alt text for the leader text, but when I go to the items within the set, the alt text option does not pop up for the images there. We need to add alt text to items that are within sets.

## Steps to reproduce

1. https://go.screenpal.com/watch/cZQbbHVS61y 

## Testing environment

* Username of the user who experienced the problem: Laken
* URL where the problem occured: 
* Browser name and version: Chrome
* User machine: Mac

## QA Steps 

[To be completed by Coko]

## Scheduling

[To be completed by Coko]

* Dependencies: [provide issue numbers]
* Development estimate: [provide in hours]

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Dione Mentis (`DioneMentis`)
> **Date:** 2024-09-23T08:49:36.311Z

`vilfranchel`  `lakenvita`  the audio didn't come through in the video, however I can see that you open a question from the set that is in a "published" state. You'll need to unpublish the question before you can make edits to it. Please try this and confirm.

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Lenszie Vilfranche (`vilfranchel`)
> **Date:** 2024-09-23T13:37:38.915Z

Correction: Yes, the items in the video are published but it is happening with unpublished items as well. Also, on the published ones, I am still able to edit/add alt text within the leader text but not the items within the set.

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Grigor Malo (`grgml`)
> **Date:** 2024-10-14T08:35:28.635Z

`vilfranchel`  I tried this in our testing site and things are working as expected. The only state where you can edit the content of an item is "In production". "Published" and "unpublished" are read-only states, where unpublishing simply removes an item from the discover page. If you want to edit an item that was previously published you have to first unpublish it, and then move it to production by clicking "Edit" in the actions bar (top right).

The leading text of a set behaves differently. The editors can edit its content anytime, even when it contains published items. You never really "unpublish" a set, you simply edit it and the changes will show up in all related items.

So to edit a published item: unpublish, then click edit, then publish again
To edit the leading text of a set: simply have the required access rights (be an editor) and visit the "Edit" tab in the set's page.

I hope this helps.

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Laken Vita (`lakenvita`)
> **Date:** 2024-10-14T19:24:59.130Z

`grgml`  I just tested this out and it works. Thank you!
